config/config.txt

```diff
@@ -342,8 +342,11 @@ NOTIFY_NEW_PLAYER_ACCOUNT_AGE 1
 ##	Requires database
 #PANIC_BUNKER
 
+## If a player connects during a bunker with less then or this amount of living time (Minutes), we deny the connection
+#PANIC_BUNKER_LIVING 90
+
 ## The message the Panic Bunker gives when someone is rejected by it
-#PANIC_BUNKER_MESSAGE Sorry, but the server is currently not accepting connections from never before seen players.
+#PANIC_BUNKER_MESSAGE Sorry, but the server is currently not accepting connections from players with less then 90 minutes of living time.
```

Comment:
```suggestion
#PANIC_BUNKER_MESSAGE Sorry, but the server is currently not accepting connections from players with less than 90 minutes of living time.
```

Though, it'd be great if this was a format string instead. As in, `Sorry, but the server is currently not accepting connections from players with less than %minutes% minutes of living time.`, which was then replaced automagically.

Reply:
That's true, though as I understand it defines don't play nice with text insertion like that. I guess I could do text parsing when it's loaded, but I'm not sure that would  go well

Reply:
No, just do `replacetext` before sending `PANIC_BUNKER_MESSAGE`.

Reply:
aight, I can do that